src/coreclr/pal/src/arch/loongarch64/context2.S

```diff
@@ -218,8 +295,85 @@ LOCAL_LABEL(Done_CONTEXT_INTEGER):
     andi  $t3, $t1, (1 << CONTEXT_FLOATING_POINT_BIT)
     beqz  $t3, LOCAL_LABEL(Done_CONTEXT_FLOATING_POINT)
 
+    andi $t3, $t1, CONTEXT_LASX
+    bnez $t3, LOCAL_LABEL(Store_CONTEXT_LASX)
+    andi $t3, $t1, CONTEXT_LSX
+    bnez $t3, LOCAL_LABEL(Store_CONTEXT_LSX)
```

Comment:
From what I can see at line 226-234, CONTEXT_ContextFlags would be `CONTEXT_FULL` at this point, how is `CONTEXT_LASX` and `CONTEXT_LSX` relate to `CONTEXT_FULL`?

IMO it should be detecting CPU feature first, then save registers accordingly, and at last set the corresponding bit in CONTEXT_ContextFlags so that the saved CONTEXT can be loaded correctly in `RtlRestoreContext`.